src/components/modals/TradePreviewModalGP.vue

```diff
@@ -1,70 +1,189 @@
 <template>
-  <BalModal
-    :show="open"
-    @close="onClose"
-    :title="$t('previewTradeTransactions')"
-  >
+  <BalModal show @close="onClose" :title="$t('previewTrade')">
     <div>
-      <div
-        class="-mx-4 p-4 flex items-center border-b border-t dark:border-gray-700"
-      >
-        <BalAssetSet
-          :addresses="[addressIn, addressOut]"
-          :size="30"
-          :width="55"
-        />
-        <div class="flex flex-col">
-          <div class="font-bold">
-            {{ fNum(amountIn, 'token') }} {{ symbolIn }} ->
-            {{ fNum(amountOut, 'token') }} {{ symbolOut }}
+      <BalCard noPad class="relative mb-6">
+        <template v-slot:header>
+          <div class="w-full p-3 border-b bg-gray-100 rounded-t-lg text-sm">
```

Comment:
There is this weird gap between the card border and the contents:
<img width="141" alt="Screenshot 2021-07-05 at 19 06 36" src="https://user-images.githubusercontent.com/2406506/124506725-2fc99280-ddc4-11eb-95c7-ab3b73be1021.png">
One option is to remove the internal rounded border and hide any overflow.
```suggestion
      <BalCard noPad class="relative mb-6 overflow-hidden">
        <template v-slot:header>
          <div class="w-full p-3 border-b bg-gray-100 text-sm">
```